plugin-server/src/cdp/cdp-consumers.ts

```diff
@@ -145,6 +145,8 @@ abstract class CdpConsumerBase {
                     value: Buffer.from(JSON.stringify(x.value)),
                     key: x.key,
                     waitForAck: true,
+                }).catch((reason) => {
+                    status.error('⚠️', `failed to produce message: ${reason}`)
```

Comment:
```suggestion
                    status.error('💩', `failed to produce message: ${reason}`)
```

Reply:
I'll leave further refinements (and waiting for the corresponding CI runs) up to you as a follow-up change